docs/agents-and-ingestion-tools/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Agents and Ingestion Tools
+nav_order: 100
+has_children: false
+has_toc: false
+---
+
+# Agents and Ingestion Tools
+
+Historically there have been multiple popular agents and ingestion tools that work with Elasticsearch. Some of which are Beats, Logstash, Fluentd, FluentBit, and Open Telemetry. OpenSearch aims to continue to support a broad set of agents and ingestion tools, but not all have been tested or have explicitly added OpenSearch compatibility. To make getting started with OpenSearch easier a [set version YML configuration](https://github.com/opensearch-project/OpenSearch/issues/693) is being added so that you can set OpenSearch 1.x clusters to report open source Elasticsearch 7.10.2. This will enable it to connect with tools that do version checking while need OpenSource compatibilities are added to your tools.
+
+
+## Compatibility Matrices
+
+So far we have built compatibility matrices for OpenSearch, Elasticsearch OSS 7.x, Open Distro for Elasticsearch (ODFE) 1.x, Logstash OSS 7.x, and Beats OSS 7.x. **This page is a living document. If there are other versions or software you would like to add to the page okease [submit a PR](https://github.com/opensearch-project/documentation-website/edit/main/docs/agents-and-ingestion-tools/index.md) that includes them and feel free to add yourself to the Credits and Thanks you at the bottom of the page. We greatly appreciate the help!** 
```

Comment:
" If there are other versions or software you would like to add to the page okease submit a PR that includes them and feel free to add yourself to the Credits and Thanks you at the bottom of the page. We greatly appreciate the help!"<- check grammar